Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM talis/ubuntu:1404-latest
+
+MAINTAINER Malcolm Landon "ml@talis.com"
+
+ENV DEBIAN_FRONTEND noninteractive
+
+ARG git_oauth_token
+ARG persona_oauth_client
+ARG persona_oauth_secret
+
+RUN apt-get update && apt-get upgrade -y
+
+RUN apt-get install -y --force-yes curl apt-transport-https && curl -L http://apt.talis.com:81/public.key | sudo apt-key add - && apt-get update
+
+# Install php and ant
+RUN apt-get install --no-install-recommends -y \
+		curl ca-certificates \
+		php5-cli \
+		php5-dev \
+		php5-xdebug \
+		php5-curl \
+		php5-json \
+		php-pear \
+    ant \
+    git
+
+# Install redis and cli, staring the service
+RUN apt-get install --no-install-recommends -y \
+    redis-tools \
+    redis-server
+
+# Install composer
+RUN curl https://getcomposer.org/installer | php -- && mv composer.phar /usr/local/bin/composer && chmod +x /usr/local/bin/composer
+
+RUN composer config -g github-oauth.github.com $git_oauth_token
+
+# Tidy up
+RUN apt-get -y autoremove && apt-get clean && apt-get autoclean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN mkdir -p /var/talis-php
+COPY . /var/talis-php
+
+RUN echo 'export PERSONA_TEST_HOST=https://staging-users.talis.com' >> /root/.bashrc
```

Comment:
Normally we would define the environment within a docker compose file to avoid baking in the credentials into the image. The docker compose file wouldn't include the values as they would be within the user's bashrc file. If the docker image was pushed it would take our credentials with the image.

Reply:
This isn't a blocker, but it would be a improvement.

Reply:
This is following the same pattern Nad put in for the echo client. We aren't currently pushing this docker image, it's just for local development. I think we can improve this in the future if we ever intend to push the image.